captured-training-yolov8-classification/picsellia/docker_run_training_yolov8_classification.py

```diff
@@ -1,18 +1,47 @@
 from picsellia import Client
 from picsellia.types.enums import AnnotationFileType
 from pycocotools.coco import COCO
-from utils import _move_files_in_class_directories, get_experiment
+from utils import _move_files_in_class_directories, get_experiment, prepare_datasets_with_annotation
 from ultralytics import YOLO
 import os
 import numpy as np
 from sklearn.metrics import classification_report, confusion_matrix
 import shutil
+from picsellia.exceptions import ResourceNotFoundError
 from evaluator.yolo_evaluator import ClassificationYOLOEvaluator
 
 experiment = get_experiment()
 
 dataset_list = experiment.list_attached_dataset_versions()
-if len(dataset_list) == 2:
+
+if len(dataset_list) == 3:
+    try:
+        train_set = experiment.get_dataset(name="train")
+    except Exception:
+        raise ResourceNotFoundError("Found 3 attached datasets, but can't find any 'train' dataset.\n \
+                                            expecting 'train', 'test', ('val' or 'eval')")
```

Comment:
we keep only eval here as well?

Reply:
maybe keep eval only, it will be clearer